app-agent/src/bin/relay_server.rs

```diff
@@ -0,0 +1,80 @@
+use alumet::{agent::AgentBuilder, static_plugins};
+use app_agent::{
+    agent_util, init_logger,
+    options::{cli, config, AgentModifier},
+};
+use clap::{Parser, Subcommand};
+
+type AppConfig = config::CommonArgs;
+
+fn main() {
+    let plugins = static_plugins![plugin_relay::server::RelayServerPlugin, plugin_csv::CsvPlugin];
```

Comment:
lint is different from other files

Reply:
the formatting? It follows the rules of `rustfmt`, just like the other files. The line is short enough here.